glue_jupyter/tests/images/py311-test-visual.json

```
@@ -1,5 +1,5 @@
 {
-  "glue_jupyter.bqplot.image.tests.test_visual.test_contour_units[chromium]": "53ed6a698cd0fe879f92a7c428e14878e80b2640a8e09d69bb82c3847041bfb9",
+  "glue_jupyter.bqplot.image.tests.test_visual.test_contour_units[chromium]": "19517fbbea156e9569a2e6367e926e9f301bb27e8d2f600b42cf4577aa9064d4",
```

```suggestion
  "glue_jupyter.bqplot.image.tests.test_visual.test_contour_units[chromium]": "94a99faec5500df042edde00e9cdef58e230d68e9a762c1cd0ed0499eb0de83f",
```
different matplotlib/bqplot versions on CircleCI?